.release-please-manifest.json

```diff
@@ -1,4 +1,6 @@
 {
-  "wnfs": "0.1.9",
-  "wasm-wnfs": "0.1.9"
+  "wnfs": "0.1.10",
+  "wasm-wnfs": "0.1.9",
```

Comment:
Ha. Naming issues. We need to make this consistent.
This should stop publishing.